src/Bot.js

```diff
@@ -44,10 +57,27 @@ class Bot {
     reviveChannels(date) {
         this.channels.forEach(channel => {
             if (date.getTime() - channel.updated.getTime() > channel.delay * 1000) {
-                channel.callback();
+                let source = this.chooseSourceFor(channel);
+                channel.callback(source ? source.getMessage() : '');
```

Comment:
maybe `source ? source.getMessage() : undefined`?

Reply:
Yea I was going to add a default source instead, this is a temporary measure

Reply:
👍